lib/devise/models/rememberable.rb

```diff
@@ -66,6 +66,10 @@ def remember_expires_at
         self.class.remember_for.from_now
       end
 
+      def extend_remember_period
```

Comment:
@lucasmazza Need a delegation for this case. Because that configurations methods generations are included on the "ClassMethods" of the module. The the instance objects does not have access.